sympy/functions/special/spherical_harmonics.py

```diff
@@ -262,6 +262,24 @@ def Ynm_c(n, m, theta, phi):
 
     Ynm, Znm
 
+    Examples
+    ========
+    >>> from sympy import Znm, Symbol, simplify
+    >>> from sympy.abc import n,m
```

Comment:
Add a blank line after `======`
And spaces around commas.

Reply:
Done.